openapi3.yaml

```diff
@@ -124,6 +124,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error'
+  /records/find/versions:
```

Comment:
add test
1. for valid request
2. for invalid request

Reply:
valid request exists => 'should return 200 and array of exists record versions catalog'